dyn_em/module_big_step_utilities_em.F

```diff
@@ -2532,9 +2531,18 @@ SUBROUTINE w_damp( rw_tend, max_vert_cfl,max_horiz_cfl, &
    integer :: total
    REAL :: msfuxt , msfxffl
    
+! LJW BEGIN
```

Comment:
@louiswicker 
Lou,
There are a number of these locations that are denoted `! LJW BEGIN` and `! LJW END`. During the source development process, it is understandable that this is an easy way to find "your" code. In this final version though, we would like those removed.